README.md

```diff
@@ -139,6 +144,13 @@ They also serve as examples to help build custom probes yourself.
 | ---------- | -----------
 | acl_stat_anomalies | Report on acl rule matches that exceed user defined thresholds |
 
+## EVPN
+
+| Probe | Description
+| ---------- | -----------
+| evpn | Monitors EVPN Telemetry including expected remote VTEP count |
```

Comment:
Remove this (evpn_type3).. as we've merged this into one. @huttmf would you mind refreshing probes..

Reply:
Removed